src/components/images/EditImage.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import LocalApi from "../../apis/LocalApi";
+import { Form, Button, Container } from "react-bootstrap";
+
+class EditImage extends Component {
+  state = {
+    selectedFile: null,
+    lot: 0,
+    unitNumber: "",
+    productDescription: ""
+  };
+
+  //When loading this page, retrive the data for this particular apartment from backend
+  componentDidMount() {
+    LocalApi.get(`${process.env.REACT_APP_BASEURL}/images/`)
+      .then(response => {
+        for (let i = 0; i < response.data.length; i++) {

```

Comment:
This is a nice workaround, but I think, we need an extra API method that shows a single image, e.g.
```
LocalApi.get("/images/" + this.props.match.params.id)
  .then(r => this.setState({...}))
```
...

Reply:
We need to discuss this one today :)